java/src/main/resources/template.properties

```diff
@@ -186,3 +186,25 @@ pubsubtogcs.gcs.output.data.format=
 dataplex.gcs.bq.target.dataset=<bq target dataset>
 dataplex.gcs.bq.save.mode="errorifexists"
 dataplex.gcs.bq.incremental.partition.copy="yes"
+
+# Kafka To BQ
+## Kafka servers
+kafka.bq.bootstrap.servers=<kafka broker list>
+## Kafka topics
+kafka.bq.topic=<kafka topic names>
+## BigQuery output dataset
+kafka.bq.dataset=<output bigquery dataset>
+## BigQuery output table
+kafka.bq.table=<output bigquery table>
+## GCS bucket name, for storing temporary files
+kafka.bq.temp.gcs.bucket=<gcs bucket name>
+## GCS location for maintaining checkpoint
+kafka.bq.checkpoint.location=<gcs bucket location maintains checkpoint>
```

Comment:
since you have supplied value it will be always nonEmpty in code.
Remove <...> for values which must be user supplied.

Reply:
Made requested changes